client/my-sites/media/index.js

```diff
@@ -28,6 +28,15 @@ export default function() {
 			clientRender
 		);
 
+		page(
+			'/media/:domain/:mediaId',
+			siteSelection,
+			navigation,
+			mediaController.media,
+			makeLayout,
+			clientRender
+		);
+
```

Comment:
if we change the route here to
'/media/:filter(this-post|images|documents|videos|audio)?/:domain/:mediaId',

we can also catch all of the filter views